gdc_ng_models/models/misc.py

```diff
@@ -17,12 +17,11 @@
 class FileReport(Base):
     __tablename__ = 'filereport'
 
-    id_seq = Sequence("filereport_id_seq", metadata=Base.metadata)
-    id = Column('id', Integer, primary_key=True, server_default=id_seq.next_value())
+    id = Column('id', BigInteger, primary_key=True, autoincrement=True)
     node_id = Column('node_id', Text, index=True)
     ip = Column('ip', String)
     country_code = Column('country_code', String, index=True)
-    timestamp = Column('timestamp', DateTime, server_default="now()")
+    timestamp = Column('timestamp', DateTime, server_default="now()", primary_key=True)
```

Comment:
This line had a preexisting bug that you can see in the `\d` output I posted above:
```suggestion
    timestamp = Column('timestamp', DateTime, server_default=func.now(), primary_key=True)
```
also need to import `func` from `sqlalchemy`.

gdcapi [sets](https://github.com/NCI-GDC/gdcapi/blob/master/gdcapi/download/report.py#L74) an explicit `timestamp` when creating a `FileReport`, so I don't think it broke anything, but it's still misleading.